Update support-sla.md (#4299)
SoftwareAG/c8y-docs: content/service-terms/service-level-bundle/support-sla.md

* **Critical Incidents**: Customer’s problem has a significant business impact; however, operations can continue in a restricted fashion. The Product is usable but severely limited. There is no acceptable workaround available. Customer is experiencing a significant loss of service.
* **Standard Incidents**: Customer’s problem has some business impact. The Product is usable and causes only minor inconvenience. It may be a minor Error, documentation Error, or incorrect operation of the Product, which does not significantly impede the operation of the Product.

### Support services

The services provided vary by support level as highlighted in the following table:

| **Service**                          | **Bronze**      | **Silver**                 | **Gold**                 |
| ------------------------------------ | --------------- | -------------------------- | ------------------------ |
| **Support Portal access for ATCs**   | 24/7            | 24/7                       | 24/7                     |

> | **Crisis Phone**                     | 9x5*            | 24/7                       | 24/7                     |

Reviewer: ```suggestion
| **Crisis phone**                     | 9x5*            | 24/7                       | 24/7                     |
```

| **Number of ATCs**                   | 3               | 7                          | Unlimited                |
| **Prioritized queuing**              | No              | No                         | Yes                      |

> | **Ticket Escalation**                | No              | Yes                        | Yes                      |

Reviewer: ```suggestion
| **Ticket escalation**                | No              | Yes                        | Yes                      |
```

*\* During Business Hours.*

| **Response Times**\*     | **Bronze**     | **Silver**             |  **Gold**             |
| ------------------------ | -------------- | ---------------------- | --------------------- |
| **Crisis - initially**   | 1 Business Day | 1 hour                 | 30 minutes            |
|        **- updates**     |                | Once per Business Hour | Once per hour         |
| **Critical - initially** | 1 Business Day | 4 Business Hours       | 2 hours               |
|        **- updates**     |                | Once per Business Day  | Once per Business Day |
| **Standard - initially** | 1 Business Day | 1 Business Day         | 1 day                 |
|        **- updates**     |                | Twice per week         | Twice per week        |
| **Escalated Tickets**    | Not available  | Once per Business Day  | Once per Business day |

*\* Provider and Customer can mutually agree on a different schedule on a per-ticket basis. This will generally happen after the initial investigations have been performed and a resolution is being implemented.*

The services are defined as follows:

> * **Crisis Phone**: The support telephone number is available in the Support Portal. Telephone support is provided in English only.

Reviewer: ```suggestion
* **Crisis phone**: The support telephone number is available in the Support Portal. Telephone support is provided in English only.
```

* **Response times**: Customer will receive an initial human response to a new support ticket within the defined initial response times. Follow-up communication times of updates to existing support tickets are targeted but not guaranteed.
* **Resolution plan for Crisis:** Within the first 4 hours, Provider aims to provide a concrete resolution plan, detailing the intended solution, a workaround, or a documented action plan with timelines.
* **Follow-the-sun:** For Gold customers, Provider actively hands over Crisis tickets between global regions (EMEA, APJ, AME) to ensure 24/7 progress.
* **Number of ATCs**: Restrictions to the number of ATCs apply per Customer, not per contract. Customer may contract for additional ATCs.
* **Prioritized queuing**: Support tickets are prioritized ahead of other support incidents of the same severity level but lower support level.

> * **Ticket escalation:** If a support ticket requires extra attention due to a serious change in business impact (e.g., an approaching deadline or increased severity), Customer can raise an escalation request via the Support Portal or email. Provider will review the new context and will ensure high visibility on its internal dashboards.

Reviewer: ```suggestion
* **Ticket escalation:** If a support ticket requires extra attention due to a serious change in business impact (for example, an approaching deadline or increased severity), Customer can raise an escalation request via the Support Portal or email. Provider will review the new context and will ensure high visibility on its internal dashboards.
```

* When reaching Support by telephone, Customer is to provide the incident/ticket number so that work on the incident can commence.
* Support has no obligation to solve the Customer’s issue within the response or any other time frame.

### Customer responsibilities

* Customer assigns Authorized Technical Contacts (ATCs) and communicates any changes to the list of ATCs to Provider.
* Customer's ATC is responsible for cooperating with Provider's Support and providing necessary information to reproduce, troubleshoot and resolve the experienced issue.
* When a support ticket is submitted by an ATC to Provider's Support Portal, Customer authorizes Provider, for the purposes of troubleshooting and resolving such issue, to access Customer’s cloud environment for the duration of the submitted support ticket.

> * Customer must explain the business impact in the Support ticket. Explaining why an issue is severe (e.g., "Production line stopped", "Upcoming go-live at risk", "Data loss imminent") allows Provider to validate the priority and allocate the right resources immediately.

Reviewer: ```suggestion
* Customer must explain the business impact in the Support ticket. Explaining why an issue is severe (for example, "Production line stopped", "Upcoming go-live at risk", "Data loss imminent") allows Provider to validate the priority and allocate the right resources immediately.
```
